### PR TITLE
feat : added style for card component

### PR DESCRIPTION
**Pull request by raman1999**
image
image
image

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+.card-price {
+    font-size: 1.1em;
+}
+
+.card-desc {
+    padding: 6px 0;
+}
+
+.btn-dismiss {
+    padding:  2px 6px 2px 6px;
```
Comment (vivekmehto) [inline components/card.css:69]: You can use shorthand property padding: top/bottom left/right;

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    border-radius: unset;
+    font-size: 0.7em;
+}
+
+.card-badge:after {
+    content: "";
+    position: absolute;
+    left: 100%;
+    top: 0;
+    border-width: 10px 5px;
```
Comment (vivekmehto) [inline components/card.css:55]: You can use border shorthand property.

**Review by vivekmehto — COMMENTED** [review]
one suggestion you can use shorthand property. Everything seems perfect to me and all variations are awesome.

File: components/card.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+.card-horizontal {
+    justify-content: space-between;
+    max-width:35rem;
+}
+
+.card-img {
+    width: 100%;
+    min-width: 15rem;
+    height:30vh;
+    border-radius: 1rem;
```
Comment (vaibhav-mahalle) [inline components/card.css:31]: Here you are using align self without display flex. Try adding display flex if you are using flex properties

**Review by vaibhav-mahalle — COMMENTED** [review]
Everything else looks good

File: components/card.css (hunk 3 above)
Comment (raman1999) [reply components/card.css:31]: Thanks for previewing. I am using .card-img inside .card-horizontal so flex is added there

File: components/card.css (hunk 2 above)
Comment (raman1999) [reply components/card.css:55]: Thanks for previewing. Here specifically width needed to change that's why used this

**Pull request merged**